samples/jobs/jobs.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {google} = require('googleapis');
+const sampleClient = require('../sampleclient');
+
+const PROJECT_ID = process.env.GOOGLE_CLOUD_PROJECT;
```

Comment:
Instead of this, please get the project Id in the async function by doing:

```js
const projectId = await google.auth.getProjectId();
```

Reply:
Sure!